### DISCUSSION TITLE
How is Vercel shipping the Next.js docs?

### DISCUSSION THREAD

**Discussion by devjiwonchoi** [vercel/next.js#54936] [Help] [answered]
### Summary

How are Next.js docs built with custom tags like `<AppOnly />` and `<PageOnly />` rendered on https://nextjs.org/docs ? 

My intention is to create an unofficial localization of the docs for the community.

P.S. If there's already an internalization going on somewhere, please let me know, thank you.

**Comment by devjiwonchoi** [comment]
@leerob Respectfully requesting for an answer.

**Comment by leerob** [comment, marked as answer]
Hey! The source code for rendering the docs isn't open source, just the docs content in this repo. We'd like to get to doing multiple languages for the docs in the future.

https://nextjs.org/docs/community/contribution-guide

**Reply by devjiwonchoi** [reply]
> We'd like to get to doing multiple languages for the docs in the future.

@leerob Sorry to bother you, but can you confirm whether it is allowed to initiate a localization as an unofficial, disclaiming relationship with Vercel and the Next.js team?

The project will preserve the current format of the `docs` directory to be compatible when the transfer is ready.

**Reply by leerob** [reply]
As long as it's clear it's a community effort, there shouldn't be any issue with it 👍

**Comment by ShravanSunder** [comment]
@leerob do you know if the docs app will eventually be made open source?
